turbo/src/main/kotlin/dev/hotwire/turbo/views/TurboWebChromeClient.kt

```diff
@@ -6,15 +6,16 @@ import android.webkit.WebChromeClient
 import android.webkit.WebView
 import dev.hotwire.turbo.session.TurboSession
 
+@Suppress("MemberVisibilityCanBePrivate")
```

Comment:
Is this to suppress the privacy lint check on `authority()`? We want to keep that public?